.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+# Describe what this PR does #
+
+Provide some context for the reviewer
+
+## Is there anything that requires special attention ##
```

Comment:
@humblec request addition of a section along the lines of "Provide any external context for the change", to accommodate things **like** Kubernetes links that explain why a change is required for example, or CSI spec related changes that may need to be pointed to, or generic golang related practices that may need to be pointed to.

Asking for this, as a lot of PRs assume familiarity with other eco-systems, and it would be nice to provide links to relate to them, than assume the same, as it helps those without the eco-system details to understand it better.

Hence, suggesting to provide it as an added section may make folks provide the same.

Reply:
Done. Or used the same word in above.